src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.scss

```diff
@@ -12,6 +12,12 @@
   &::placeholder {
     color: $grey;
   }
+  &:focus::-webkit-input-placeholder {
+    color: transparent;
+  }
+  &:focus::-moz-placeholder {
+    color: transparent;
+  }
```

Comment:
1) In general, we shouldn't be writing vendor prefixes — we have postcssPresetEnv, which is supposed to do it for us (https://github.com/Ensembl/ensembl-client/blob/c4aea08e45e2d76f4f17f57c5b3437fe197da7fb/src/ensembl/.storybook/webpack.config.js#L46)

2) From the CSS perspective, would 
```
&:focus {
  &::placeholder {
    ... rules
  }
}
```
make more sense?

3) Rather than doing this with CSS, why don't we remove placeholder text in BrowserRegionField component (https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.tsx#L143) when the input is focused?

Reply:
> Rather than doing this with CSS, why don't we remove placeholder text in BrowserRegionField component (https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/browser/browser-region-field/BrowserRegionField.tsx#L143) when the input is focused?

We need the placeholder when the region field input is not focussed

Reply:
onFocus: remove placeholder
onBlur: show placeholder

Reply:
On the other hand, you are right, if this is easier done with CSS, let's do it with CSS.

Reply:
@azangru Updated css. Have a look and let me know.

Reply:
Looks good.